# HG changeset patch
# Parent  210f442e3f50d9add6b94b0f7f9feba591d9c7b9
Set the verbosity to INFO by default

Debug is very rarely needed even when actually debugging something.

diff --git a/core/lightsd.c b/core/lightsd.c
--- a/core/lightsd.c
+++ b/core/lightsd.c
@@ -55,7 +55,7 @@
 struct lgtd_opts lgtd_opts = {
     .foreground = false,
     .log_timestamps = true,
-    .verbosity = LGTD_DEBUG
+    .verbosity = LGTD_INFO
 }; 
 
 struct event_base *lgtd_ev_base = NULL;
